src/features/PostFeedPage/PostFeedPage.jsx

```diff
@@ -89,128 +114,151 @@ export const PostFeedPage = () => {
-                    </li>
-                    <li
-                      onClick={(e) => {
-                        setShowEmojis(true);
-                        if (
-                          e.target.childNodes.length === 1 &&
-                          e.target.innerText !== 'Emojis'
-                        ) {
+                        className='grow focus:outline-none font-light text-txt-secondary-color'
+                        placeholder='Write something here'
+                        value={postInputForm.content}
+                        onChange={(e) =>
                           setPostInputForm({
                             ...postInputForm,
-                            content: postInputForm.content + e.target.innerText,
-                          });
-
-                          setShowEmojis(false);
+                            content: e.target.value,
+                          })
                         }
-                      }}
-                      className='relative flex items-center gap-3 bg-secondary-background py-2 px-3 rounded-md cursor-pointer'
-                    >
-                      <img
-                        className='h-6 w-6'
-                        src='https://res.cloudinary.com/donqbxlnc/image/upload/v1652278871/Sunglasses_Emoji_be26cc0a-eef9-49e5-8da2-169bb417cc0b_grande_tz0jya.png'
-                        alt='emojis'
+                        type='text'
                       />
-                      <p className='text-primary text-sm font-semibold'>
-                        Emojis
-                      </p>
-                      {showEmojis ? (
-                        <div
-                          ref={emojiContainerRef}
-                          className='absolute w-48 p-4 flex flex-wrap justify-center items-center gap-1 rounded-lg  bg-secondary-background '
-                        >
-                          {emojiLib.map((el) => {
-                            return (
-                              <span
-                                key={el}
-                                className='cursor-pointer text-2xl'
-                              >
-                                {el}
-                              </span>
-                            );
-                          })}
-                        </div>
-                      ) : null}
-                    </li>
-                  </ul>
+                    </div>
+                    {postInputForm.pic ? (
+                      <div className='relative'>
+                        <img src={postInputForm.pic} alt='post pic' />
+                        <i
+                          onClick={() =>
+                            setPostInputForm({ ...postInputForm, pic: '' })
+                          }
+                          className='absolute top-1 right-1 text-4xl text-txt-hover-color cursor-pointer fas fa-times-circle'
+                        ></i>
+                      </div>
+                    ) : null}
```

Comment:
I feel that you can probably just use `&&` instead of using the ternary operator (`?` and `:`) here.